tools/extensions/BUILD

```diff
@@ -9,17 +9,21 @@ envoy_package()
 py_binary(
     name = "generate_extension_db",
     srcs = ["generate_extension_db.py"],
-    data = [
-        "@com_github_bazelbuild_buildtools//buildozer:buildozer",
-    ] + envoy_all_extensions(),
-    python_version = "PY3",
-    srcs_version = "PY3",
-    visibility = ["//visibility:public"],
 )
 
 py_binary(
     name = "generate_extension_rst",
     srcs = ["generate_extension_rst.py"],
     data = [":generate_extension_db"],
-    visibility = ["//visibility:public"],
+)
+
+exports_files([
```

Comment:
this has been added to prevent regression - should be removed before landing this PR